config/config/sc-config.yaml

```
@@ -1,4 +1,7 @@
 global:
+  ## Compliantkubernetes-apps version.
+  ## You need to be at the exact branch for the version number to match.
+  ## 'any', can be used when running on non-official release.
```

```suggestion
  ## Compliantkubernetes-apps version.
  ## Use version number if you are exactly at a release tag.
  ## Otherwise use full commit hash of current commit.
  ## 'any', can be used to disable this validation.
```
I'd prefer something like this. Also update the other file.